packages/cf-deploy-config-inquirer/src/prompts/prompt-helpers.ts

```diff
@@ -49,8 +49,8 @@ export async function getCfSystemChoices(destinations?: Destinations): Promise<C
 export async function fetchBTPDestinations(log?: Logger): Promise<Destinations | undefined> {
     if (isAppStudio()) {
         const destinations = await listDestinations();
-        log?.warn(t('warning.btpDestinationListWarning'));
         return destinations;
     }
+    log?.warn(t('warning.btpDestinationListWarning'));
```

Comment:
@kjose90 @longieirl You know that adding logging in this way, rather than allowing a reference to a logger be passed into the inquirer, means we cant integrate with the YUI logger? Please see the `ui5-library-inquirer` for an example otherwise.